packages/backend/server/src/core/auth/resolver.ts

```diff
@@ -101,14 +95,6 @@ export class AuthResolver {
       throw new LinkExpired();
     }
 
-    const config = await this.config.runtime.fetchAll({
```

Comment:
why remove password limit

Reply:
`auth.changePassword` uses `user.updateUser` which includes all the validations